src/java.base/share/classes/javax/crypto/CryptoPermissions.java

```diff
@@ -229,10 +229,7 @@ CryptoPermissions getMinimum(CryptoPermissions other) {
         // For each algorithm in this CryptoPermissions,
         // find out if there is anything we should add into
         // ret.
-        Enumeration<String> thisKeys = this.perms.keys();
```

Comment:
There are a couple other places where you could do the same enhanced for loop with arrays:  Current lines: 107, 254, 287.

Reply:
There are many places in javax crypto which iterate over an array. I think it's worth to create a separate issue